frozendict/frozendict.pyi

```diff
@@ -1,6 +1,57 @@
-try:
-    from ._frozendict import *
-except ImportError:
-    from .core import *
+import collections
+from _typeshed import Self
+from collections.abc import Iterable, Iterator, Mapping
+from typing import Any, Generic, NoReturn, Tuple, TypeVar, Union, overload
 
-FrozenOrderedDict = frozendict
+_KT = TypeVar("_KT")
+_VT = TypeVar("_VT")
+
+def frozendict_or(
+    self: Mapping[_KT, _VT], other: Mapping[_KT, _VT]
```

Comment:
self can't be a generic immutable map, must be a frozendict

Reply:
Oh, yeah, I was looking at the Python implementation of `frozendict_or`. I'll update it.